Palace.xcodeproj/project.pbxproj

```diff
@@ -4429,9 +4433,10 @@
 				CLANG_ENABLE_MODULES = YES;
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Palace/SimplyE.entitlements;
-				CODE_SIGN_IDENTITY = "iPhone Distribution";
-				CURRENT_PROJECT_VERSION = 101;
-				DEVELOPMENT_TEAM = 88CBA74T8K;
+				CODE_SIGN_IDENTITY = "Apple Development";
```

Comment:
We can't use this, `fastlane` needs `iPhone Distribution` with `Ad hoc 2` profile

Reply:
Ah, meant to revert this, thanks.